lib/venue.rb

```diff
@@ -1,2 +1,25 @@
 class Venue
-end
+  attr_reader :name, :capacity, :patrons
+  def initialize(name, capacity, patrons = [])
+    @name = name
+    @capacity = capacity
+    @patrons = patrons
+  end
+  def add_patron(patron)
+    @patrons += [patron]
```

Comment:
As this is written you are adding/reseting `@patrons`. How do you just add something to an array?

https://ruby-doc.org/core-2.5.0/Array.html#:~:text=Adding%20Items%20to,4%2C%205%2C%206%5D